manifests/a/ActiveState/KomodoEdit/12.0.1/ActiveState.KomodoEdit.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: ActiveState.KomodoEdit
+PackageVersion: 12.0.1
+PackageLocale: en-US
+Publisher: ActiveState Software Inc.
+PackageName: Komodo Edit 12

```

Comment:
```suggestion
PackageName: ActiveState Komoto Edit
```
![image](https://user-images.githubusercontent.com/11600822/120943127-ef271c80-c724-11eb-8fb0-12557628823b.png)

Reply:
I am not sure about this edit because please take a look at #16012, which naming convention should we use for the PackageName - with the author prefix or not?

Reply:
Most of the time the PackageName should be similar or exact shown from Control Panel to allow other people to upgrade via `winget upgrade` because that command uses Control Panel to find the application.

Reply:
Ah, that makes absolute sense, thanks for the edit and for the explanation. 😀